_posts/blog/2018-09-05/2018-09-05-troubleshooting-and-how-to-help-with-development-on-openbazaar.markdown

```diff
@@ -0,0 +1,110 @@
+
+**3) Community Discussion on [Reddit](https://reddit.com/r/openbazaar)**
+
+Reddit allows for topical, threaded discussions and lets users discuss all sorts of things. Some leading topics in the OpenBazaar subreddit include: new stores, decentralized ecommerce news, cryptocurrency mechanics and issues which are hard to track down. The discussion on Reddit shifts over time but its persistence gives a good archive of past issues and thought-provoking conversation. Participating in the OpenBazaar subreddit can be anonymous and has no barrier to posting a question or comment short of creating a Reddit account. Most anything even slightly related to OpenBazaar can be posted here.
+
+**4) Real-time Discussion on [Slack](https://openbazaar.org/slack) and [Telegram](https://t.me/OpenBazaarGroup)**
+
+For high-bandwidth conversation with other users and contributors, our team and community congregate in these 2 discussion forums that function a lot like real-time chatrooms. Some areas of these communities focus on 3rd party search providers, moderators and escrow, new ideas being tested on OpenBazaar, as well as offering each other help and support on common issues. Slack, byfar the most popular of the bunch, has the most regular activity as OB1 and many other developers are engaged there. Telegram is not quite as active but is a nice addition to Slack were people can pop in to ask quick questions about OpenBazaar or determine their next steps troubleshooting an issue.
+
+**5) Development on [Github](https://github.com/openbazaar)**
+
+Github contains the technical discussions and objectives of our various projects. Each repository contains a history of commits from previous contributors, a list of issues that represent pending work, and in some cases further technical documentation which is coupled to the software. This is where the rubber meets road as far as where improvements are discussed and added. It's safe to assume if a problem or issue is not being tracked in Github, it is not being worked on.
+
+### Reporting The Problem
+
+Okay, so what happens when you have exhausted all avenues of personal research and are still stumped? OB1 funnels all code changes through [Github's issue tracker](https://github.com/OpenBazaar/openbazaar-desktop/issues). When providing a bug report, including enough information in order to reproduce--and then fix--the problem is the primary objective. 
+
+**Here is a brief checklist of useful information that should be included in the report:**
+
+1. Mention your **operating system (OS) and version**. If the OS type and version is unknown for your computer, you can visit [http://whatsmyos.com](http://whatsmyos.com/) to help. (ex: Windows 10, MacOS 10.12.16, Ubuntu 16.04)
+   
+2. Mention your **OpenBazar software version**. This is indicated in the top right corner of the About screen. (ex: client 2.2.4, server 0.12.3)
```

Comment:
I'd add instructions here on how to get to the about screen, like "...the About screen, which can be launched from the navigation menu in the upper right."